src/lib/pages/backoffice/EItem/EItemDetails/EItemMetadata/EItemMetadata.js

```diff
@@ -19,6 +20,12 @@ export default class EItemMetadata extends Component {
         name: 'Available urls',
         value: <UrlList urls={eitemDetails.metadata.urls} />,
       },
+      {
+        name: 'Identifiers',
+        value: (
+          <IdentifierRows identifiers={eitemDetails.metadata.identifiers} />
```

Comment:
did you try with empty identifiers field ? or with the field not existing on the object?

Reply:
(fixed in `26c037c`)